web/thesauruses/c/17/io.json

```diff
@@ -0,0 +1,278 @@
+{
+    "meta": {
+        "language": "c",
+        "language_version": "17",
+        "language_name": "C",
+        "structure": "io"
+    },
+    "concepts": {
+        "write_line": {
+            "code": "printf(\"Format Specifier\", var1, var2, …., varn);",
+            "name": "Write a line to the console"
+        },
+        "write_line_with_new_line" : {
+            "code": "printf(\"Format Specifier\", var1, var2, …., varn);",
+            "name": "Write a line to the console with a return character"  
+        },
+        "read_line": {
+        "code": "scanf(\"Format Specifier\", &var1, &var2, …., &varn);",
+        "comment": "'&' is used to send memory address of variable(pass by reference) since changes to variable is being made(i.e. storing values) but not in printf since it is only accessing those values not changing it",
+        "name": "Read a line of user input from the console"
+        },
+        "read_char": {
+            "code": "getchar()",
+            "name": "Read a single char of user input from the console"
+            },
+       "clear_console_cutput": {
+            "code": [
+                "#include<stdio.h>",
+                "#include<stdlib.h>",
+                "int main(){",
+                "system(\"cls\");",
+                "printf(\"hello\");",
+                "return 0;",
+                "}"
```

Comment:
I just realized this is a whole program and not just the part that clears the screen. (Also why are you adding more to the screen with the print statement?)

I think `system` also runs a system command, and while `cls` is one that clears the screen, that's only for Windows. It's `clear` on Mac/Linux. So I don't think you can do this statement here anyway.

I'd just make this "not-implemented" since it's not a built-in thing in the language.